xbmc/video/ContextMenus.cpp

```diff
@@ -119,7 +120,17 @@ static void SetPathAndPlay(CFileItem& item)
     item.SetPath(item.GetVideoInfoTag()->m_strFileNameAndPath);
   }
   item.SetProperty("check_resume", false);
-  g_application.PlayMedia(item, "", PLAYLIST_NONE);
+
+  CFileItemPtr movieItem((std::make_shared<CFileItem>(item)));
```

Comment:
Exception safe? 

=> http://en.cppreference.com/w/cpp/memory/shared_ptr/make_shared => "May throw std::bad_alloc or any exception thrown by the constructor of T. If an exception is thrown, this function has no effect."

And why is it more efficient? Don't get this.